ui/component/claimPreview/index.js

```diff
@@ -22,21 +22,23 @@ import { doResolveUri } from 'redux/actions/claims';
 import { doCollectionEdit } from 'redux/actions/collections';
 import { doFileGet } from 'redux/actions/file';
 import { selectBanStateForUri } from 'lbryinc';
-import { makeSelectIsActiveLivestream } from 'redux/selectors/livestream';
 import { selectShowMatureContent } from 'redux/selectors/settings';
 import { makeSelectHasVisitedUri } from 'redux/selectors/content';
 import { makeSelectIsSubscribed } from 'redux/selectors/subscriptions';
 import ClaimPreview from './view';
 import formatMediaDuration from 'util/formatMediaDuration';
+import { selectActiveChannelClaim } from 'redux/selectors/app';
 
 const select = (state, props) => {
   const claim = props.uri && selectClaimForUri(state, props.uri);
+  const { claim_id: channelId } = selectActiveChannelClaim(state) || {};
```

Comment:
Can use `selectActiveChannelId` to avoid looking up the claim and reducing back down to an ID.